Sources/NetworkRequestHelper.swift

```diff
@@ -188,11 +188,12 @@ class NetworkRequestHelper {
 
     /// Asserts that the correct number of network requests were seen for all previously set network request expectations.
     /// - Parameters:
+    ///   - ignoreUnexpectedRequests: A Boolean value indicating whether unexpected requests should be ignored. Defaults to `true`.
+    ///   - timeout: The time interval to wait for network requests before timing out. Defaults to ``TestConstants/Defaults/WAIT_NETWORK_REQUEST_TIMEOUT``.
     ///   - file: The file from which the method is called, used for localized assertion failures.
     ///   - line: The line from which the method is called, used for localized assertion failures.
-    /// - SeeAlso:
-    ///     - ``setExpectationForNetworkRequest(url:httpMethod:)``
-    func assertAllNetworkRequestExpectations(ignoreUnexpectedRequests: Bool = true, file: StaticString = #file, line: UInt = #line) {
+    /// - SeeAlso: ``setExpectationForNetworkRequest(url:httpMethod:)``
```

Comment:
Do we have a ``setExpectationForNetworkRequest(url:httpMethod:)``
or should use
- ``setExpectationForNetworkRequest(url:httpMethod:expectedCount:file:line:)``